dockers/docker-config-engine-stretch/Dockerfile.j2

```diff
@@ -1,5 +1,5 @@
 {% from "dockers/dockerfile-macros.j2" import install_debian_packages, install_python_wheels, copy_files %}
-FROM docker-base-stretch
+FROM {{ docker_config_engine_stretch_load_image }}
```

Comment:
Where do these variables defined?

Reply:
This variable is populated from '_LOAD_DOCKERS' variable in the slave.mk file. 

$(eval export $(subst -,_,$(notdir $($*.gz_PATH)))_load_image=$(shell printf "$(call docker-load-image-get,$(subst $(SPACE),\n,$(patsubst %.gz,%,$(call expand,$($*.gz_LOAD_DOCKERS)))))\n" | awk '!a[$$0]++'))

Reply:
I think this is a good change, it used _LOAD_DOCKER variable in dockerfile.j2. actually, I do not quite understand why it is LOAD_DOCKERS, it should be LOAD_DOCKER.

Reply:
It is a LOAD_DOCKERS(not LOAD_DOCKER) which contains the name of the dependent docker. Eg: 
FILE:  docker-snmp.mk
$(DOCKER_SNMP)_LOAD_DOCKERS += $(DOCKER_CONFIG_ENGINE_BUSTER)